practice/src/js/helpers/httpClient.js

```diff
@@ -0,0 +1,104 @@
+class HttpClient {
+  constructor(options = {}) {
+    this._baseURL = options.baseURL || '';
+    this._headers = options._headers || {};
+  }
+
+  /**
+   * Send requests and receive JSON from the server.
+   * @param {string} endpoint - The API endpoint.
+   * @param {Object} options - Options for the request.
+   * @returns {Promise<any>} - A promise with the response from the server.
+   */
+  async _fetchJSON(endpoint, options = {}) {
+    const response = await fetch(this._baseURL + endpoint, {
+      ...options,
+      headers: this._headers,
+    });
+
+    if (!response.ok) throw new Error(response.statusText);
+
+    if (options.parseResponse !== false && response.status !== 204)
```

Comment:
Can you use shorthand !options.parseResponse instead of `options.parseResponse !== false`?

Reply:
Yes